spec/factories/users.rb

```diff
@@ -5,6 +5,7 @@
     last_name { Faker::Artist.name }
     password { Faker::Color.color_name }
     role { :default }
+    activated { true }
```

Comment:
Updated factories to reflect new field. Set as true for testing sake but default is false.